maths/maths.py

```diff
@@ -0,0 +1,112 @@
+
+# package maths
+# Implement some 
+# trigonometric algorithm
+
+Pi = 3.14159265358979323846
+
+
+def _abs(value):
+    # >>> _abs(-10)
+    # 10
+    # >>> _abs(10)
+    # 10
+    return -value if value < 0 else value
+
+
+def _power(value, n):
+    # >>> _power(-10, 2)
+    # 100
+    # >>> _power(10, 2)
+    # 100
+    result = 1
+
+    if n == 0:
+        result = 1
+    else:
+        for i in range(n):
+            result = result * value
+
+    return result
+
+
+# newton iteration method
+def _sqrt(value):
```

Comment:
```suggestion
def _sqrt(value):
    """
    >>> _sqrt(9)
    3
    >>> _sqrt(25)
    5
    """
```
The docstring must appear at the top of the function before any code.